vignettes/decorate-module-output.Rmd

```diff
@@ -12,16 +12,21 @@ vignette: >
 
 ## Introduction
 
-`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis. 
-This document outlines the customization options available for modifying the output of `teal` modules. 
-You will learn how to use `teal_transform_module()` to modify and enhance the objects created by `teal::modules()`, 
+`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis.
+This document outlines the customization options available for modifying the output of `teal` modules.
+
+You will learn how to use `teal_transform_module()` to modify and enhance the objects created by `teal::modules()`,
 enabling you to tailor the outputs to your specific requirements without rewriting the original module code.
 
 ## Decorators
 
-In programming, **decoration** refers to the process of modifying an object while preserving its original class. For instance, given an object `x` of class `"my_class"`, a function `foo(x)` is considered a **decorator function** if it modifies `x` and returns an object that retains the same class. In this context, `x` is referred to as the **decorated object**, and `foo()` is the **decorator function** or **decorator**. Decorators can perform a variety of operations, such as adding new methods or modifying data, while ensuring the object remains compatible with its original usage.
+In programming, **decoration** refers to the process of modifying an object while preserving its original class.
+For instance, given an object `x` of class `"my_class"`, a function `foo(x)` is considered a **decorator function** if it modifies `x` and returns an object that retains the same class.
+In this context, `x` is referred to as the **decorated object**, and `foo()` is the **decorator function** or **decorator**. Decorators can perform a variety of operations, such as adding new methods or modifying data, while ensuring the object remains compatible with its original usage.
```

Comment:
```suggestion
In this context, `x` is referred to as the **object to be decorated**, and `foo()` is the **decorator function** or **decorator**. Decorators can perform a variety of operations, such as adding new methods or modifying data, while ensuring the object remains compatible with its original usage.
```

"decorated object" sounds like it's the result of `foo(x)`

Reply:
I think that the while paragraph has no meaningful value. Referring to `decoration` in programming is patronizing. I think word "decorate" is self exploratory I could even write that "Decoration is a process of changing the look of the object without altering its fundamental function". I would simply skip this section and leave the line below "In the context of `teal` applications, decoration...". WDYT?

Reply:
makes sense to me.

Reply:
Done in af32a06b5990e4f6cbd18680cd088697781ac141